mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -113,8 +123,10 @@ def _update_output_signatures(self):
         -------
 
         """
-        self.batch_output_signature = tf.TensorSpec(shape=(self.batch_size, 3), dtype=tf.float64)
-        self.ensemble_output_signature = tf.TensorSpec(shape=(self.data_range, 3), dtype=tf.float64)
+        self.batch_output_signature = tf.TensorSpec(shape=(self.batch_size, 3),
+                                                    dtype=tf.float64)
```

Comment:
`self.dtype` should be a property that can be set to `dtype=tf.float32` or `float64` so we can change the accuracy of the calculators for performance reasons

Reply:
Sure. When was this introduced? Because I saw the code when I was updating things but had no idea it was introduced. This is the kind of thing we need to have in the dev docs or something as without information about it it will be hard to always enforce.

Reply:
Let's open an issue to address this.